test/defi/erc20RewardPool/rules/ContributionRules.test.ts

```diff
@@ -453,5 +453,19 @@ describe('ContributionRules', function () {
 
       expect(await contract.getMaxGlobalMultiplier(other)).to.be.equal(30);
     });
+    it('should not be able to set setERC721MultiplierLimit > 1000', async function () {
+      const {contractAsAdmin} = await ContributionRulesSetup();
+
+      await expect(
+        contractAsAdmin.setERC721MultiplierLimit(1500)
+      ).to.be.revertedWith('ContributionRules: invalid newLimit');
+    });
+    it('should not be able to set setERC1155MultiplierLimit > 1000', async function () {
+      const {contractAsAdmin} = await ContributionRulesSetup();
+
+      await expect(
+        contractAsAdmin.setERC1155MultiplierLimit(1500)
+      ).to.be.revertedWith('ContributionRules: invalid newLimit');
+    });
   });
 });
```

Comment:
missing tests for LockRules setTimelockClaim, etc

Reply:
done -> https://github.com/thesandboxgame/sandbox-smart-contracts/pull/662/commits/3dc9dd501c550803c8b8051fa6a42917069a24b5